### PR TITLE
Update Vagrant configuration and documentation

### LINKED ISSUE #245: Criar ambiente Vagrant para avaliação do IDGB
Precismos criar um ambiente [Vagrant](https://www.vagrantup.com/) que permita a avaliação simples do IDGB sem ter maiores conhecimentos de Linux, Buildout e as outras tecnologias envolvidas numa instalação de produção.

Precisamos que fique documentado como criar esse ambiente para poder atualizar ele cada vez que um novo release for gerado.

### PR DESCRIPTION
**Pull request by hvelarde**
This is based on the following:

* https://github.com/plone/training/tree/master/plone_training_config
* https://www.vagrantup.com/docs/index.html

closes https://github.com/plonegovbr/brasil.gov.portal/issues/245

### CODE REVIEW

**Comment by hvelarde** [comment]
we can still enhance this by creating a special `vagrant.cfg` for buildout, but I think is good enough to be used.

**Comment by hvelarde** [comment]
I think the configuration is still overcomplicated and prone to errors; I'll try to refactor it the next iteration following the same logic we have in Travis.

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+Configuração Vagrant para IDG
+=============================
+
+.. contents:: Conteúdo
+   :depth: 2
+
+Introdução
+==========
+
+`Vagrant <https://www.vagrantup.com/>`_ é uma ferramenta para construir e ferenciar ambientes de máquinas virtuais en um workflow simples.
```
Comment (rodfersou) [inline README.rst]: "ferenciar" => "gerenciar"
"en" => "em"

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+Abre uma janela em su web browser e aponte a ``localhost:8080``.
+
+.. figure:: https://raw.githubusercontent.com/plonegovbr/portal.vagrant/master/up-and-running.png
+    :align: center
+    :height: 768px
+    :width: 1024px
+
+O Plone está disponível e pronto para criação de sites IDG.
+
+Para deter o servidor Zope presione ``Ctrl+C``:
```
Comment (rodfersou) [inline README.rst]: "deter" => "parar" ou "desligar"

**Pull request merged**